social-media-research-database/app/projects/ManageProjectPostsModal.tsx

```
@@ -0,0 +1,685 @@
+              <tbody>
+                {associatedPosts.map((post) => (
+                  <tr
+                    key={`${post.datetime}-${post.username}-${post.social_name}`}
+                  >
+                    <td>{timeFormat(post.datetime)}</td>
+                    <td>{post.username}</td>
+                    <td>{post.social_name}</td>
+                    <td>{post.text?.slice(0, 40)}...</td>
+
+                    {selectedProject.fields.map((field) => (
+                      <td>{post.field_results?.[field]}</td>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)